CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
```

Comment:
Do we want to leave the unreleased header in the released CHANGELOG or is this a bug?

Reply:
I think it was on purpose so it is there ready to add to

Reply:
And to avoid everyone from committing the same line after each release which causes merge conflicts